Align on use of listener (#26953)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/downloads/oncreated/index.md

- `function`

>   - : The function called when this event occurs. This function will be passed this argument:

Author: ```suggestion
  - : The function called when this event occurs. This function is passed this argument:
```
